### ISSUE TITLE
Dev: stages

### ISSUE THREAD

**Issue by ivan-kleshnin** [AntonKilk/FA_app#11]
# List of pages and widgets

## Client "Mobile" App

- [ ] 1. Layout
  - [ ] 1.1 Common Sections
  - [ ] 1.2 Logo
  - [ ] 1.3. Fixed Top Menu
- [ ] 2. "Info" / "About" / "Help" pages
- [ ] 3. "Measurement"  page
  - [ ] 3.1 Animation
  - [ ] 3.2 "Result" screen with "Notes" form
- [ ] 4. "History of Measurements" page
  - [ ] 4.1 Table / list
  - [ ] 4.2 Action icons
- [ ] 5. "Measurement Review" page
- [ ] 6. "Edit Measurement" page with form

## Admin "Desktop" App

- [ ] 10. Layout
  - [ ] 10.1 Common Sections
  - [ ] 10.2 Logo
- [ ] 11. "Login" page with form
- [ ] 12. "All Measurements" page with filters
- [ ] 13. "Measurement Review" page

**Comment by ivan-kleshnin** [comment]
Let's first confirm that the above is a full enumeration of all we want to have/to do. 
Then let's create a separate issue for each point we'll take into work. I propose to add resource links to that pages so we don't bloat this list to the point we lose the wholistic view. Or to the comments here but not to the above enumeration where each point should remain a one-liner.

**Comment by ivan-kleshnin** [comment]
I attempted to order the above list in the way I imagine the development. 

We can make all the coding first and then add styling OR we can add styles in parallel.
@dmitrij-schmidt which approach would you prefer? @AntonKilk your opinion?
